### ISSUE TITLE
关于mobiletnetv2_voc里train.prototxt的bn

### ISSUE THREAD

**Issue by smallhaolzh** [eric612/MobileNet-YOLO#238]
我看到train.prototxt里的BatchNorm是没有另外显式设置lr_mult和decay_mult为0的，这样的话lr_mult和decay_mult默认为1，可是caffe里的BatchNorm层训练的时候的参数(mean, std)应该从数据获得，而不应该靠训练更新？请问这里是不是有什么我没考虑到的地方？

**Comment by eric612** [comment]
預設會參考是train還是test
https://github.com/eric612/MobileNet-YOLO/issues/209#issuecomment-554891547

**Comment by eric612** [comment]
And the learning rate you can see this 
https://stackoverflow.com/questions/41608242/how-should-batchnorm-layer-be-used-in-caffe
